prowler/providers/gcp/gcp_provider.py

```diff
@@ -41,6 +38,10 @@ def __init__(
             # If not projects were input, all accessible projects are scanned by default
             self.project_ids = accessible_projects
 
+        # Set Default Project ID if not set in credentials
+        if not self.default_project_id:
+            self.default_project_id = self.project_ids[0]
```

Comment:
Why are you getting the first project as the default? Can we skip that part?

Reply:
Because the output file name needs the default_project_id.